tests/apm/monitoring-addSubscriber-001.phpt

```diff
@@ -26,7 +26,6 @@ class MySubscriber implements MongoDB\Driver\Monitoring\CommandSubscriber
     }
 }
 
-CLEANUP( STANDALONE );
```

Comment:
Does this not mean we're not cleaning up data? Maybe I misunderstand.

Reply:
These instances of `CLEANUP()` were redundant, as it was already done in `SKIPIF`. See: https://github.com/mongodb/mongo-php-driver/blob/0273c79be43ef55fcc861b7b4992602739245f67/tests/apm/monitoring-addSubscriber-001.phpt#L4